test/runtests.py

```diff
@@ -211,6 +224,38 @@ def _wait_for_queue_record(self):
             if Queue.query.filter(Queue.name == consumer.queue_name).first():
                 break
 
+    def _wait_for_binding_record(self, queue_name, exchange_name, routing_key):
+        """Wait until a biding has been added to the database"""
+        consumer = self._create_passive_consumer()
+        attempts = 0
+        while attempts < self.QUEUE_RECORD_CHECK_ATTEMPTS:
+            attempts += 1
+            if attempts > 1:
+                time.sleep(self.QUEUE_RECORD_CHECK_PERIOD)
+            self.guardian.monitor_queues(pulse_management.queues())
+            if Binding.query.filter(Binding.queue_name == queue_name,
```

Comment:
Might as well follow your earlier format and put the first argument on its own line.